### PR TITLE
Merge rc from 2021.3.3 to master

### PR DESCRIPTION
**Pull request by seanbudd**
This should be a merge commit, not a squash merge.

Adds the changelog from 2021.3.2 and 2021.3.3

### CODE REVIEW

**Comment by seanbudd** [comment]
I am a little concerned about the old commits showing up here, but it appears that the changes are what they should be

File: source/appModuleHandler.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 from fileUtils import getFileVersionInfo
 
 # Dictionary of processID:appModule pairs used to hold the currently running modules
-runningTable: Dict[int, AppModule] = {}
+runningTable: Dict[int, "AppModule"] = {}
```
Comment (lukaszgo1) [inline source/appModuleHandler.py:42]: Why this change is here? Note that it is unnecessary on master since type hints in `appModuleHandler` are evaluated lazily.

File: source/appModuleHandler.py [hunk 2]
```diff
@@ -107,7 +107,7 @@ def getAppModuleForNVDAObject(obj):
 	return getAppModuleFromProcessID(obj.processID)
 
 
-def getAppModuleFromProcessID(processID: int) -> AppModule:
+def getAppModuleFromProcessID(processID: int) -> "AppModule":
```
Comment (lukaszgo1) [inline source/appModuleHandler.py:110]: Same here.

File: source/appModuleHandler.py (hunk 1 above)
Comment (seanbudd) [reply source/appModuleHandler.py]: These were necessary on `rc`, and had to be added. I've removed this from the merge.

**Comment by feerrenrut** [comment]
This shouldn't be merged as is.
